dashboard/app/views/levels/editors/fields/_long_instructions.html.haml

```diff
@@ -13,7 +13,26 @@
     %div{ style: 'background-color: rgb(239, 239, 239); padding-top: 12px' }
       ~ f.text_area :long_instructions, rows: 4
 
+- if @level.is_a? GamelabJr
+  = f.label :instructions_icon,'Instructions Icon'
+  %p Please refer to #{link_to 'Sprite Lab Static Assets', 'https://github.com/code-dot-org/code-dot-org/tree/staging/apps/static/spritelab'} for available assets, then enter the name of the image to use as the instructions icon for this level.
```

Comment:
not sure how likely this is, but we could run into a situation where the assets listed in `apps/static/spritelab/` on the staging branch aren't available on levelbuilder yet. we could point this URL to the production (or levelbuilder) branch instead of staging?